Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -585,7 +585,7 @@ internal void OnClientConnectInternal(NetworkMessage netMsg)
         {
             if (LogFilter.Debug) { Debug.Log("NetworkManager:OnClientConnectInternal"); }
 
-            string loadedSceneName = SceneManager.GetSceneAt(0).name;
+            string loadedSceneName = SceneManager.GetActiveScene().name;
```

Comment:
great find. could we do this as separate PR first?

Reply:
split to another PR